docs/usage/dns_names.md

```diff
@@ -25,95 +25,121 @@ spec:
 spec:
   rules:
-    - host: test.ingress.my-dns-domain.com
-      http:
-        paths:
-          - backend:
-              service:
-                name: my-service
-                port:
-                  number: 9000
-            path: /
-            pathType: Prefix
-  tls:
-    - hosts:
-        - test.ingress.my-dns-domain.com
-      secretName: my-cert-secret-name
+  - host: special.example.com
+    http:
+      paths:
+      - backend:
+        serviceName: amazing-svc
+        servicePort: 8080
 ```
 
-For a Service (it must have the type `LoadBalancer`) this looks like this:
+Keep in mind that ingress resources are ignored unless an ingress controller is set up. Gardener does not provide an ingress controller by default. See the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) and [Gardener documentation](https://github.com/gardener/documentation/blob/master/website/documentation/guides/applications/service-access/_index.md#loadbalancer-vs-ingress) for more details.
+
+### Request DNS records for service type LoadBalancer
 
+Example for an annotated Service (it must have the type `LoadBalancer`) resource:
 ```yaml
 apiVersion: v1
 kind: Service
 metadata:
+  name: amazing-svc
   annotations:
+    # Let Gardener manage external DNS records for this Service.
+    dns.gardener.cloud/dnsnames: special.example.com
+    dns.gardener.cloud/ttl: "600"
     dns.gardener.cloud/class: garden
-    dns.gardener.cloud/dnsnames: my.subdomain.for.some.domain.cloud
-  name: my-service
-  namespace: default
 spec:
+  selector:
+    app: amazing-app
   ports:
-    - port: 80
-      protocol: TCP
-      targetPort: 80
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
   type: LoadBalancer
 ```
 
-The *dnsnames* annotation accepts a comma-separated list of DNS names, if
-multiple names are required.
-
-For an Ingress, the DNS names are already declared in the specification.
-Nevertheless the *dnsnames* annotation must be present. Here a subset of the 
-DNS names of the ingress can be specified. If DNS names for all names are
-desired, the value `all` can be used.
+#### Creating a DNSEntry resource explicitly
+It is also possible to create a DNS entry via the Kubernetes resource called `DNSEntry`:
+```yaml
+apiVersion: dns.gardener.cloud/v1alpha1
+kind: DNSEntry
+metadata:
+  annotations:
+    # Let Gardener manage this DNS entry.
+    dns.gardener.cloud/class: garden
+  name: special-dnsentry
+  namespace: default
+spec:
+  dnsName: special.example.com
+  ttl: 600
+  targets:
+  - 1.2.3.4
+```
 
-If one of the accepted DNS names is a direct subname of the shoot's ingress
-domain, this is already handled by the standard wildcard entry for the ingress
-domain. Therefore this name should be excluded from the *dnsnames* list in the
-annotation. If only this DNS name is configured in the ingress, no explicit 
-DNS entry is required, and the DNS annotations should be omitted at all.
```

Comment:
Why should this useful information be dropped?

Reply:
You're right, probably deleted that by error